### ISSUE TITLE
[Bug]: injectValuesForCustomLayout(): Argument #1 ($layout) must be of type array, null given, called in 

### ISSUE THREAD

**Issue by mattiasghodsian** [pimcore/pimcore#15027]
### Expected behavior

Ability to add DataObject without issues

### Actual behavior

Trying to create a new data object and am encountering an error 
![image](https://user-images.githubusercontent.com/20265517/234220204-804a7d9a-597f-4f97-bd8d-bf5469dc51bf.png)
![image](https://user-images.githubusercontent.com/20265517/234220146-0337703b-a5f2-4eff-a383-74596063c730.png)

```
Timestamp: Tue Apr 25 2023 10:29:46 GMT+0200 (Central European Summer Time)
Status: 500 | Internal Server Error
URL: /admin/object/get
Message: Pimcore\Bundle\AdminBundle\Controller\Admin\DataObject\DataObjectController::injectValuesForCustomLayout(): Argument #1 ($layout) must be of type array, null given, called in /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php on line 574
Trace: 
in /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php:583
#0 /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php(574): Pimcore\Bundle\AdminBundle\Controller\Admin\DataObject\DataObjectController->injectValuesForCustomLayout(NULL)
#1 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(163): Pimcore\Bundle\AdminBundle\Controller\Admin\DataObject\DataObjectController->getAction(Object(Symfony\Component\HttpFoundation\Request), Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#2 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(75): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#3 /var/www/html/vendor/symfony/http-kernel/Kernel.php(202): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#4 /var/www/html/public/index.php(36): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#5 {main}
```

### Steps to reproduce

I have the docker container setup on a VPS

**Comment by torqdev** [comment]
Hi @mattiasghodsian, having trouble replicating this issue locally. Are you able to to send the steps to reproduce this error? Also are you able to replicate this error on the [development environment](https://demo.pimcore.fun/admin)?

**Comment by mattiasghodsian** [comment]
@torqdev This is how i installed it

Nginx proxy
```
server {
    listen 80;
    server_name pimcore.webbhuset.com;

    location / {
        proxy_pass http://localhost:86;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
    }

}
```

```shell
docker run -u $(id -u):$(id -g) --rm -v $(pwd):/var/www/html pimcore/pimcore:php8.2-latest composer create-project pimcore/skeleton .
```
Changed the nginx port `80` to `86` 
```shell
docker-compose up -d
```
Installed pimcore and setup mysql connection
```shell
docker-compose exec php vendor/bin/pimcore-install --mysql-host-socket=db --mysql-username=pimcore --mysql-password=pimcore --mysql-database=pimcore
```
Set www-data owner of `var` in the project so we don't get any issues
```shell
docker-compose exec php chown -R www-data:www-data /var/www/html/var/
```

**Comment by mattiasghodsian** [comment]
@torqdev This is how i installed it

Nginx proxy
```
server {
    listen 80;
    server_name pimcore.site.com;

    location / {
        proxy_pass http://localhost:86;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
    }

}
```

```shell
docker run -u $(id -u):$(id -g) --rm -v $(pwd):/var/www/html pimcore/pimcore:php8.2-latest composer create-project pimcore/skeleton .
```
Changed the nginx port `80` to `86` 
```shell
docker-compose up -d
```
Installed pimcore and setup mysql connection
```shell
docker-compose exec php vendor/bin/pimcore-install --mysql-host-socket=db --mysql-username=pimcore --mysql-password=pimcore --mysql-database=pimcore
```
Set www-data owner of `var` in the project so we don't get any issues
```shell
docker-compose exec php chown -R www-data:www-data /var/www/html/var/
```

**Comment by fashxp** [comment]
can you please share your class definition?

**Comment by dvesh3** [comment]
Reproducible with empty class definition. It is not a valid case however we should still take care of the exception.

**Comment by dvesh3** [comment]
Fixed by #15201
